tests/QueryObjectTest.php

```diff
@@ -28,6 +28,11 @@ public function setUp(): void
         $this->queryObject = new SimpleQueryObject('simples');
     }
 
+    private function replaceWhitespace(string $string) {
```

Comment:
@actionm  I don't get why do we need this?

Reply:
I have updated the PR comment with the full explanation.